src/gsi/setupdbz.f90

```diff
@@ -360,97 +616,604 @@ subroutine setupdbz(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,radardbz_d
+                 rdBZr = ten * log10(Zer)
+                 rdBZs = ten * log10(Zes)
+                 rdBZg = ten * log10(Zeg)
+    
+!                find dqr/ddBZ, dqs/ddBZ, dqg/ddBZ (used in inner loop routine)
+!                Jacobian used for TLM and ADM
+                 if ( l_use_cvpqx ) then
+!                   rain
+                    if ( qrexp > 10E-7_r_kind .and. qscalar(P_nr) > 20._r_kind ) then
+                       jqr_num = (2*720*rho**2_r_kind*qrexp**(2_r_kind-cvpqx_pval)/&
+                                 (pi**2_r_kind*rhor**2_r_kind*qscalar(P_nr)))*ten**12_r_kind*ten**6_r_kind
+                       if (l_cvpnr) then
+                          jqnr_num = (-720*rho**2_r_kind*qrexp**2_r_kind/&
+                                     (pi**2_r_kind*rhor**2_r_kind*qscalar(P_nr)**(1_r_kind+cvpnr_pval)))&
+                                     *ten**12_r_kind*ten**6_r_kind
+                       else
+                          jqnr_num = (-720*rho**2_r_kind*qrexp**2_r_kind/&
+                                     (pi**2_r_kind*rhor**2_r_kind*qscalar(P_nr)**2_r_kind))&
+                                     *ten**12_r_kind*ten**6_r_kind
+                       end if
+                    else
+                       jqr_num = 10E-8_r_kind
+                       jqnr_num = 10E-8_r_kind
+                    endif
+    
+!                   snow
+                    if (qsexp > 10E-7_r_kind) then
+                       if ( T1D > 274.15_r_kind .and. iqrges > 1.E-12_r_kind) then
+                          jqs_num= (1.47E+05_r_kind)*(1000._r_kind)**2.67_r_kind*(2.67_r_kind)*(qsexp)**(2.67_r_kind-cvpqx_pval)
+                       else
+                          jqs_num= alpha_const_tm_dry_snow*a_dry_snow_tm*b_dry_snow_tm &
+                                   *((rho)**b_dry_snow_tm)*((oams)**b_dry_snow_tm)*((qsexp)**(b_dry_snow_tm-cvpqx_pval))
+                       end if
+                    else
+                       jqs_num = 10E-8_r_kind
+                    endif
+    
+!                   graupel/hail
+                    if ( qgexp > 10E-7_r_kind ) then
+                       if ( T1D > 274.15_r_kind .and. iqrges > 1.E-12_r_kind) then
+                          jqg_num=Cg_wet*((rho)**1.75_r_kind)*Pg_wet*((qgexp)**(Pg_wet-cvpqx_pval))
+                       else
+                          jqg_num=Cg_dry*((rho)**1.75_r_kind)*Pg_wet*((qgexp)**(Pg_wet-cvpqx_pval))
+                       end if
+                    else
+                       jqg_num=10E-8_r_kind
+                    endif
+    
+                 else ! CVq
+    
+!                   rain
+                    if ( qrexp > 10E-7_r_kind .and. qscalar(P_nr) > 1._r_kind ) then
+                       jqr_num = (2_r_kind*720*rho**2_r_kind*qrexp/&
+                                 (pi**2_r_kind*rhor**2_r_kind*qscalar(P_nr)))*ten**12_r_kind*ten**6_r_kind
+                       jqnr_num = (-720*rho**2_r_kind*qrexp**2_r_kind/&
```

Comment:
Please add _r_kind to 720

Reply:
We changed the code as you suggested.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-7a9c44619cb5bc5c1e48fcc0d434a8be4257007f9f4a2c8a9cd9090ff3873135R1082-R1085